src/pages/Home/Home.tsx

```diff
@@ -1,17 +1,14 @@
 import Layout from "../../components/Layout/Layout";
+import Navigation from "../../components/Navigation/Navigation";
 
 const Home = () => {
   return (
     <Layout>
       <h1>Page Heading</h1>
       <h2>Section Heading</h2>
       <h3>Panel Heading</h3>
```

Comment:
I think you should remove this from `Home` , the component should be inside the `App` so each page doesn't need to load the component